zsh/configs/post/path.zsh

```diff
@@ -1,8 +1,10 @@
 # ensure dotfiles bin directory is loaded first
 PATH="$HOME/.bin:/usr/local/sbin:$PATH"
 
-# load rbenv if available
-if command -v rbenv >/dev/null; then
+# load ASDF, falling back to rbenv if not available
+if [ -d "$HOME/.asdf" ]; then
+  . $HOME/.asdf/asdf.sh
+elif command -v rbenv >/dev/null; then
   eval "$(rbenv init - --no-rehash)"
 fi
```

Comment:
Everything else in this file is for PATH-specific stuff. Should the version management portion be extracted to a separate file? Looks like the load order matters and is currently alphabetically by file? https://github.com/thoughtbot/dotfiles/blob/master/zshrc